### ISSUE TITLE
Site Locales Issue

### ISSUE THREAD

**Issue by aerni** [justkidding96/statamic-aardvark-seo#61]
The locales in my site config are set to `en_US.utf8` instead of simply `en_US`. This results in the following:

```html
<link rel="alternate" href="http://michaelaerni-v3.test/de" hreflang="de_DE.utf8">
<meta property="og:locale" content="en_US.utf8">
```

I'm no SEO expert, by I think that Google expects locales to be like `en_US`.

**Comment by AndrewHaine** [comment]
👍  Consulted with our SEO team, will add a filter to strip out any unwanted parts from the locale before generating the tags

**Comment by AndrewHaine** [comment]
Hi @aerni ,

Just released 2.0.3 which fixes this issue. We are now stripping out `.utf8` if it is present and then converting the locale to ISO 639-1 with a country code (the result will be something like `de-DE`)

**Comment by aerni** [comment]
Sweet!
